tensorboard/plugins/debugger/interactive_debugger_server_lib.py

```diff
@@ -83,15 +84,23 @@ def _comm_tensor_data(device_name,
   output_slot = int(output_slot)
   tf.logging.info(
       'Recording tensor value: %s, %d, %s', node_name, output_slot, debug_op)
-  tensor_dtype = str(tensor_value.dtype)
-  tensor_shape = tensor_value.shape
-  # The /comm endpoint should respond with tensor values only if the tensor is
-  # small enough. Otherwise, the detailed values sould be queried through a
-  # dedicated tensor_data that supports slicing.
-  if tensor_helper.numel(tensor_shape) < 5:
-    _, _, tensor_values = tensor_helper.array_view(tensor_value)
+  tensor_values = None
+  if isinstance(tensor_value, debug_data.InconvertibleTensorProto):
+    if not tensor_value.initialized:
+      tensor_dtype = 'Uninitialized'
+      tensor_shape = 'Uninitialized'
```

Comment:
nit: Maybe store "Uninitialized" and "Unsupported" within comments.

Reply:
I think you meant "constants" instead of "comments". Done.